lang/javascript/javascript.py

```diff
@@ -55,6 +87,7 @@ def code_state_switch():
 
     def code_state_case():
         actions.auto_insert("case :")
+        actions.key("left")
```

Comment:
Bug fix

Reply:
This kind of thing should preferably use snippets

Reply:
snippets are nice, but generally knausj repo doesn't depend on them, prob in part so it can work anywhere without snippet support.  tho i have a vague recollection that you have some impl of snippets that can work in more generic contexts using a fallback?

Reply:
Yes I have a snipped system that fall backs on text insertion. It even calculates the number of arrow key steps it has to take to get to the default tab stop. 
https://github.com/AndreasArvidsson/andreas-talon/blob/ec039ba885e41c59943a722a05883e8d9460ea68/text/snippets.py#L15

Reply:
This snippet thing sounds great, just never made the time. :)

Reply:
Neat!  Filed https://github.com/knausj85/knausj_talon/issues/848 to track